src/main/java/n3/arrays/ArrayTask7.java

```diff
@@ -0,0 +1,24 @@
+package n3.arrays;
+
+/*
+- написати програму яка рахує середнє значення елементів масиву з індексом кратним 7
+ */
+
+public class ArrayTask7 {
+    public static void main(String[] args) {
+        int [] arr = { -5, 0, 1, 2, 3, 5, 28, 6, 8, 11, 56, 87, 54, 34, 21, 9, 80, 81, 90, -100};
+        int avg = 0;
```

Comment:
Please, declare this variable at the end of the program. AS you dont need it here.

Reply:
which variable do you mean?

Reply:
if AVG - then it seems I can't declare this variable in the loop. It'll have been created during each iteration

Reply:
fixed, pls check if ok